### DISCUSSION TITLE
Compatibility with Macbook Pro 2017

### DISCUSSION THREAD

**Discussion by yiqiangjizhang** [kvic-z/SsdPmEnabler#9] [Q&A]
Hey, I have a Macbook Pro 2017 13". I have upgraded my ssd to a new 1TB. However, I faced sever hibernatioon battery consumption issues. After researching I came across this tool. I would like to ask the compatibility of it with my model. I have read the README.md and it does not list my model, but I would like to ask if there is any problem if I try it.

Thanks!

**Comment by kvic-z** [comment]
Hi

ssdpmEnabler would reduce your new SSD's idle and operational power consumptions. It should also work for 2016/17 non-touch models. To reduce hibernate/sleep power consumption, check [Q1](https://github.com/kvic-z/SsdPmEnabler/wiki/FAQ#1-how-to-reduce-battery-drain-during-sleep) in the FAQ.

Let me know how it goes. With more data points, I'm going to update the README & the table of supported models to include 2016/17 non-touch models.

Cheers

**Comment by yiqiangjizhang** [comment]
Hi @kvic-z, so I happened to try it out, i did all the installation process as shown in the README.md, however, I still face a similar idle consumption issue... :(

**Comment by kvic-z** [comment]
Wait a second.

You said you had battery drain issue when sleep/hibernate. Was that resolved? Note that battery drain when **sleep/hibernate** and battery drain **at idle** are two different things.

Do you see "kernel[0]: (SsdPmEnabler) Enabled PCIe PM on SSD" output from [here](https://github.com/kvic-z/SsdPmEnabler#confirm-ssdpmenabler-working-properly)? What is your idle current measurement shown in iStat Menus?

**Reply by yiqiangjizhang** [reply]
Hi @kvic-z !

So I think I have battery drain when sleep/hibernate and it is not yet resolved. I guess I have also battery drain at idle as well.

i checked and I do have the "kernel[0]: (SsdPmEnabler) Enabled PCIe PM on SSD" message. I do not have iStat Menus purchased :(. I can try to downloaded it to send you the report measurements

**Comment by kvic-z** [comment]
> i checked and I do have the "kernel[0]: (SsdPmEnabler) Enabled PCIe PM on SSD" message. I do not have iStat Menus purchased :(. I can try to downloaded it to send you the report measurements

That's very positive. At least idle consumption will be reduced. The trial version of iStat Menus will do for this purpose. You should see some obvious drop in idle current drawn by your SSD.

**Comment by originate0** [comment]
for anyone who is searching information for if it works on a1708 or not - yep, it works, and really well. i mean the battery life of my macbook has increased significantly.
i have an aura ssd, and had no battery draining on idle problem, but on usage my battery drain very fast.
so thank you very much, kvic!
